javascript-data-model/order-history.js

```diff
@@ -0,0 +1,57 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    totalPrice: 34.00,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    deliveredDate: 'August 8, 2020',
+    handedToResident: false,
+    title: 'JavaScript for impatient programmers',
+    hasAuthor: true,
+    returnWindowDate: 'September 7, 2020',
+    originalPrice: 31.55,
+    writtenProductReview: false
+  },
+
+  {
+    orderPlaced: 'July 19, 2020',
+    totalPrice: 44.53,
+    shipTo: 'JS Masher',
+    orderNum: '113-9984268-1280257',
+    deliveredDate: 'July 20, 2020',
+    handedToResident: true,
+    title: 'The Timeless Way of Building',
+    hasAuthor: true,
+    returnWindowDate: 'August 19, 2020',
+    originalPrice: 41.33,
+    writtenProductReview: false
+  },
+
+  {
+    orderPlaced: 'July 4, 2020',
+    totalPrice: 17.22,
+    shipTo: 'JS Masher',
+    orderNum: '114-2875557-9059409',
+    deliveredDate: 'July 7, 2020',
+    handedToResident: true,
+    title: 'Gamecube Controller Adapter',
+    hasAuthor: false,
+    returnWindowDate: 'August 5, 2020',
+    originalPrice: 15.98,
+    writtenProductReview: false
+  },
+
+  {
+    orderPlaced: 'July 3, 2020',
+    totalPrice: 138.93,
+    shipTo: 'JS Masher',
+    orderNum: '113-2883177-2648248',
+    deliveredDate: 'July 5, 2020',
+    handedToResident: false,
+    title: ['Gamecube Controller', 'The Art of Sql'],
+    hasAuthor: [false, true],
+    returnWindowDate: 'August 4, 2020',
+    originalPrice: [94.95, 33.99],
+    writtenProductReview: [false, true]
```

Comment:
In this object, these properties have a value that is an array, but in the other order objects, these properties hold a single primitive value. This format is inconsistent and will lead to errors if you actually tried to use this data model. You need to create a format that is able to be used in any variation of the data you might need to work with.